### DISCUSSION TITLE
留言时遇到问题

### DISCUSSION THREAD

**Discussion by Wu-Haitao** [walinejs/waline#793] [Q&A] [answered]
留言会出错。
但是刷新之后发现确实已经发上去了。
![屏幕截图 2022-02-03 013935](https://user-images.githubusercontent.com/59434361/152208127-70ffde91-ceee-4ddc-960b-6c63d11328c8.png)
![image](https://user-images.githubusercontent.com/59434361/152208141-3bd9fe3f-b6a8-4ad7-b132-1eeb7100312d.png)

**Comment by lizheming** [comment, marked as answer]
可以上后台看下日志，应该是通知之类的服务报错了

**Reply by Wu-Haitao** [reply]
还真是，把邮件通知关了就正常了。
感谢大佬🙏
